### ISSUE TITLE
usage

### ISSUE THREAD

**Issue by Lijingtangbo** [SUwonglab/arcsv#16]
Hi, 

Can this program be adapted for WGS of other species (easily)? If so, how to? 

Thanks in advance

**Comment by jgarthur** [comment]
Yes, any assembly in fasta format can be used. I imagine the gaps.bed files are not available for all genomes, but it is an optional file and has minimal impact on the final results.
